### PR TITLE
LIFX SmartApp Update

### PR DESCRIPTION
**Pull request by chendo**
This PR updates the LIFX SmartApp. Changes:
- Use v1 of the API
- Support multi attribute tiles
- Better handling of unreachable lights

### CODE REVIEW

File: devicetypes/smartthings/lifx-white-bulb.src/lifx-white-bulb.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- *  LIFX White Bulb
+ *  LIFX Color Bulb
```
Comment (workingmonk) [inline devicetypes/smartthings/lifx-white-bulb.src/lifx-white-bulb.groovy]: why is the white bulb being named color bulb?

**Comment by workingmonk** [comment]
looks good to start testing

File: devicetypes/smartthings/lifx-white-bulb.src/lifx-white-bulb.groovy (hunk 1 above)
Comment (chendo) [inline devicetypes/smartthings/lifx-white-bulb.src/lifx-white-bulb.groovy]: Header copy and paste fail, will fix.

Comment (chendo) [inline devicetypes/smartthings/lifx-white-bulb.src/lifx-white-bulb.groovy]: Fixed in 132c265

**Comment by workingmonk** [comment]
Hey @chendo 
We had to make Oauth changes to all our cloud connected devices, so we merged your changes as part of those changes here #291 

The code is deployed to prod in US and EU

**Comment by chendo** [comment]
@workingmonk cool, that works for me. So there's nothing we need to do on our end for EU to work? We don't have a EU SmartThings account to verify ourselves

**Comment by workingmonk** [comment]
@chendo yup, we have QA in US and UK testing on the EU accounts to make everything works perfectly

**Pull request closed without merging**